mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -332,6 +337,48 @@ func resourceComposerEnvironment() *schema.Resource {
 								},
 							},
 						},
+						"recovery_config": {
+							Type:         schema.TypeList,
+							Optional:     true,
+							AtLeastOneOf: composerConfigKeys,
+							MaxItems:     1,
+							Description:  `The recovery configuration settings for the Cloud Composer environment`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"scheduled_snapshots_config": {
+										Type:         schema.TypeList,
+										Optional:     true,
+										AtLeastOneOf: recoveryConfigKeys,
+										Description:  `The configuration settings for scheduled snapshots.`,
+										MaxItems:     1,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"enabled": {
```

Comment:
Does this field support in-place update?

Reply:
Yes it supports in-place update.